main.c

```diff
@@ -0,0 +1,17 @@
+#include <stdlib.h>
+#include <stdio.h>
+
+int main(void)
+{
+  int number;
+  int factorial = 1;
+  printf("Enter  number: \n");
+  scanf("%d", &number);

```

Comment:
QUESTION😎 
What happens if, instead of numbers, you enter the string "Some symbols, not decimal" in the command line?

Reply:
will be an incorrect value